docs/book/localization.md

```diff
@@ -323,21 +323,192 @@ in the Deployment Chapter.
+        "another french utterance {slot_name}"
+      ]
+    }
+  },
+  "speech": { // map of all in-skill speech to map of Litexa language with any available translations
+    "default speech string": { // say/reprompt string found in default Litexa files
+      "fr": "override string for FR"
+    },
+    "say statement|alternate one": { // say/reprompt with their `or` alternates are delineated by `|` characters
+      "fr": "french alternate one|french alternate two|french alternate three"
+    }
+  }
+}
+```
+
+To add your localized strings, map them from the target language to the string,
+for each default string you wish to localize. Litexa's SSML shorthand and
+interpolation rules continue to apply.
+
+:::tip Escaped SSML tags require an extra `\`
+Due to how say strings are parsed, if you have escaped SSML tags:
+
+```coffeescript
+  say "\<amazon:emotion name='excited'>Cats are cute!\</amazon:emotion>"
+```
+
+They render in `localization.json` as:
+
+```json
+"<amazon:emotion name='excited'>Cats are cute!</amazon:emotion>"
+```
+
+And you need to add the escapes back into the mapped translations with another
+backslash:
+
+```json
+"speech": {
+  "<amazon:emotion name='excited'>Cats are cute!</amazon:emotion>": {
+    "fr": "\\<amazon:emotion name='excited'>Cats are cute!\\</amazon:emotion>"
+  },
+  ...
+}
 ```
+:::
+
+A few things to observe:
+* Utterances for a language are not mapped one to one, meaning that an intent
+in the default language might have 3 utterances, and the one for fr-FR might
+have 5. You can have any number of utterances for an intent for any language.
+* Say/reprompt strings and their alternatives are mapped together. They appear
+as one entry in the file as pipe (`|`) separated strings. As such, your
+localized language can have asymmetrical alternatives. Both default and
+localized strings follow the same format.
+
+This method reads strings from only the default skill code. Because the default
+skill code is the source of truth, modifying any of the default language speech
+strings in the localization file will break the mapping. A subsequent
+`litexa localize` will show modified strings as orphaned, and re-add the
+original strings to the file, with logging output stating they are new speech
+lines. Modifying any default language intents in `localization.json` will
+behave the same way, and any changes from modifying default language utterances
+will be ignored, and overridden by `litexa localize` executions.
+
+The speech strings function as overrides in the skill. If there is no translated
+string for a given language, it will fall back to the default string.
+
+### More on `litexa localize`
+
+`litexa localize` can receive some arguments to modify its behavior. To see the
+complete list, run `litexa localize --help`.
+
+#### Cloning
+
+With cloning, the arguments for cloning your default language to French would look like:
+
+```bash
+litexa localize --clone-from default --clone-to fr
+```
```

Comment:
Would maybe tweak the example since cloning the default strings is rarely recommendable (seeing as default is already the fallback for missing translations). Would instead give the example of `litexa localize --clone-from fr-CA --clone-to fr-FR` (or some other 2 regional non-English languages). Could perhaps even add a tip that explains why this is useful (cloning one non-default language to another non-default language, to have a starting point for adjusting translations for the different region).